### PR TITLE
LPD-29534 Use current company instead of fragmentEntryLink company

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPD-29534

### CODE REVIEW

**Comment by brookedalton** [comment]
ci:test:publications-db-partition

**Comment by brookedalton** [comment]
ci:test:publications

File: modules/apps/fragment/fragment-entry-processor/fragment-entry-processor-freemarker/src/main/java/com/liferay/fragment/entry/processor/freemarker/FreeMarkerFragmentEntryProcessor.java [hunk 1]
```diff
@@ -78,7 +79,7 @@ public String processFragmentEntryLinkHTML(
 			freeMarkerFragmentEntryProcessorConfiguration =
 				_configurationProvider.getCompanyConfiguration(
 					FreeMarkerFragmentEntryProcessorConfiguration.class,
-					fragmentEntryLink.getCompanyId());
+					CompanyThreadLocal.getCompanyId());
```
Comment (pei-jung) [inline modules/apps/fragment/fragment-entry-processor/fragment-entry-processor-freemarker/src/main/java/com/liferay/fragment/entry/processor/freemarker/FreeMarkerFragmentEntryProcessor.java:82]: This doesn't seem right. How does the fragmentEntryLink from a different company get here? We should figure out where the fragment is fetched and make sure it's fetching the one with the correct companyId.

**Comment by brookedalton** [comment]
Sent to https://github.com/liferay-echo/liferay-portal/pull/15876

**Pull request closed without merging**